tests/integration/tests/user_scenarios/duplicate_same_module_version.rb

```diff
@@ -6,10 +6,15 @@
 # Init
 local_files_root_path = ENV['MANIFESTS'] || 'tests/manifests'
 
-dsc_source_module_path = 'C:\ProgramData\PuppetLabs\puppet\cache\lib\puppet_x\dsc_resources\xPSDesiredStateConfiguration'
+puppet_agent_cache_path = 'C:\ProgramData\PuppetLabs\puppet\cache\lib\puppet_x'
```

Comment:
Same thing with paths here - Beaker has helpers for this kind of stuff already based on it's defaults.  And you can always ask the agent for the value.